jmclient/jmclient/wallet.py

```diff
@@ -285,6 +346,7 @@ def save(self):
         """
         Write data to associated storage object and trigger persistent update.
         """
+        self._utxos.save(write=False)
         self._storage.save()
```

Comment:
this could simply be `self._utxos.save()`

Reply:
Done in 1925e0da95a1a1180431a4c3e89a35195f55d889